CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endif()
 
 message(STATUS "Libs version: ${FALCOSECURITY_LIBS_VERSION}")
 
-if(CMAKE_SYSTEM_NAME MATCHES "Linux" AND USE_BUNDLED_DRIVER)
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
```

Comment:
Why are we ignoring `USE_BUNDLED_DRIVER` here? :thinking:

Reply:
I think we should always configure driver_config.h, even on non-Linux systems!

Reply:
Mmmh but then, we really only need driver versions on linux systems (ie: the ones that actually build drivers).
I am not sure how to proceed :/

Reply:
This is the cleanest approach I could think of, things like moving the `driver_config.h` configuration up to this point don't make much sense, so to me this makes sense as it can be interpreted as:
- If we are compiling in linux, go configure the drivers.
- If we are not actually building the driver, cut the configuration short.

If there is any other way, I'm open to hearing it!